assets/express/src/api/index.ts

```diff
@@ -0,0 +1,21 @@
+import { DbCollection } from '@database';
+import { HealthzController, healthzModuleFactory } from './healthz';
+import { UserController,  userModuleFactory } from './user';
+
+export interface ApiRoutesDefinition extends Record<string, object> {
+  healthz: HealthzController;
+  users: UserController;
+  // todos: TodosController
+}
+
+export type ApiRoutes = UserController[keyof UserController] | HealthzController[keyof HealthzController]
```

Comment:
I don't this would scale well and I don't see a reason of typing this. This is only used in the `app.js` but all we need is a generic route type that has `method, url, requestSchema, middlewares, handler` etc.